Manual_for_achitecture_browser/index.asciidoc

```diff
@@ -0,0 +1,76 @@
+= Manual for architecture browser
+These content help you to know how you can contribute to the content of the architecture browser .
+
+== Stack specific solution
+A stack specific solution is the place where you can find all the solutions of each stack. 
+You should do each solution on a new branch and not take the master branch because that can lead to conflicts.
+
+== Fork repository
+First of all, you have to fork your own copy of devonfw/architectures to your account, by clicking on the icon "Fork".
+
+image::fork.PNG[Fork_Repository,100,100]
+
+== Open the repository on Gitpod
+The repository is already on your account. You have to click on "code" and copy the url.
+
+image::url.png[URL,300]
```

Comment:
The extension does not match the one of the file. You should use PNG or png, here and in the file name! I would use png.